.github/workflows/package.yml

```diff
@@ -10,6 +10,45 @@ on:
     - cron: |
         0 0 * * *
 jobs:
+  source:
+    name: Source
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt -y -V install \
+            qemu-user-static \
```

Comment:
Do we need this?
We don't use Docker image for arm64.

Reply:
Oh, is `qemu-user-static` installed for Docker image for arm64 at this context?
Ok, I will remove this.

Reply:
Yes.

Reply:
Thanks, fixed.